python/cudf/cudf/_libxx/table.pyx

```diff
@@ -87,6 +87,7 @@ cdef class Table:
         column_names : iterable
         """
         cdef vector[unique_ptr[column]] columns
+        print(c_tbl.get().num_columns())
```

Comment:
I guess you're still debugging?

Reply:
I had been and forgot to remove this. removing